libethereum/TransactionQueue.cpp

```diff
@@ -230,6 +222,17 @@ Transactions TransactionQueue::topTransactions_WITH_LOCK(
     return top_transactions;
 }
 
+// note - this function is currently only used when tracing is enabled
```

Comment:
I couldn't find the place where this function is used

Reply:
It is used in createBlock